src/forms/registration-popup/index.jsx

```diff
@@ -284,7 +284,7 @@ const RegistrationForm = () => {
       ...onboardingComponentContext, ...queryParams, ...payload,
     };
     payload = snakeCaseObject(payload);
-    payload.totalRegistrationTime = totalRegistrationTime;
+    payload.total_registration_time = totalRegistrationTime;
```

Comment:
```
payload.totalRegistrationTime = totalRegistrationTime;
payload = snakeCaseObject(payload);
```

Reply:
@ahtesham-quraish please take a look at this